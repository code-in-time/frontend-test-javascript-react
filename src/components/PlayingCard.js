import React, { Component } from 'react';
import PropTypes from 'prop-types';

class PlayingCard extends Component {
  render() { 
    return ( 
      <div className="card">
      <div className="card-body">
        <ul>
          <h5 className="card-title">{this.props.name}</h5>
          <li>Mass: {this.props.mass}</li>
        </ul>
      </div>
    </div>
     );
  }
}

PlayingCard.defaultProps = {
  name: '',
  mass: ''
};

PlayingCard.propTypes = {
  name: PropTypes.string,
  mass: PropTypes.string
};
 
export default PlayingCard;
