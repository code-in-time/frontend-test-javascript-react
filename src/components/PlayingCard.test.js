import React from 'react';
import renderer from 'react-test-renderer';
import PlayingCard from './PlayingCard';

it('Should test the PlayingCard.js snapshot', () => {
  const tree = renderer.create(<PlayingCard />).toJSON()
  expect(tree).toMatchSnapshot();
})
