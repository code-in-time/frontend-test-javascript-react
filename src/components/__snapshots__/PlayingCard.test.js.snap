// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test the PlayingCard.js snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card-body"
  >
    <ul>
      <h5
        className="card-title"
      >
        
      </h5>
      <li>
        Mass: 
        
      </li>
    </ul>
  </div>
</div>
`;
